app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/legacy/LiveEvent.kt

```diff
@@ -0,0 +1,58 @@
+package com.hedvig.android.feature.chat.legacy
+
+import androidx.annotation.MainThread
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.Observer
+
+// to be deleted asap when Chat gets reworked. Copied over from :app so we don't expose this everywhere in a common
+// module
+internal class LiveEvent<T> : MediatorLiveData<T>() {
```

Comment:
This, like some other files here are just duplicated from the :app module. They are inside the `legacy` package indicating that we do want to just delete them asap!